package.json

```diff
@@ -240,6 +246,18 @@
         "title": "Upload Schema to Schema Registry",
         "category": "Confluent: Schemas"
       },
+      {
+        "command": "confluent.schemas.evolve",
+        "icon": "$(plus)",
+        "title": "Evolve Schema",
+        "category": "Confluent: Schemas"
+      },
+      {
```

Comment:
New command to evolve the highest versioned schema within this the subject group.